FirebaseRemoteConfig/Tests/FakeConsole/FakeConsoleTests.swift

```diff
@@ -0,0 +1,71 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import FirebaseCore
+@testable import FirebaseRemoteConfig
+
+import XCTest
+
+class FakeConsoleTests: APITestBase {
+  override func setUp() {
+    super.setUp()
+    fakeConsole.config = ["Key1": "Value1"]
+  }
+
+  // Contrast with testUnchangedActivateWillError in APITests.swift.
+  func testChangedActivateWillNotError() {
+    let expectation = self.expectation(description: #function)
+    config.fetch { status, error in
+      if let error = error {
+        XCTFail("Fetch Error \(error)")
+      }
+      XCTAssertEqual(status, RemoteConfigFetchStatus.success)
+      self.config.activate { error in
+        if let error = error {
+          print("Activate Error \(error)")
```

Comment:
I'm guessing this should also be removed :)

Reply:
With the deprecated API whether or not there is an error on the first activate call is random depending on the state of the console - so it might be useful to keep until we can delete along with the deprecated API at the next major release.